test/dynamo/test_repros.py

```diff
@@ -3998,6 +3998,53 @@ def func3(x, y):
                 # frame_count should stay at 1.
                 self.assertEqual(cnt.frame_count, 1)
 
+    def test_flash_attn_backward_mixed_strides(self):
```

Comment:
You can import this helper from:
```Python
from torch.testing._internal.common_cuda import PLATFORM_SUPPORTS_FLASH_ATTENTION
```
```suggestion
     @skipIfRocm  
    @unittest.skipIf(
        not PLATFORM_SUPPORTS_FLASH_ATTENTION, "Platform does not supposrt fused SDPA or pre-SM80 hardware"
    )
```

In theory you shouldn't have to skip rocm but it can be pretty hit or miss if new tests work on those runners